src/components/transactions/TxDetails/TxData/DecodedData/index.tsx

```diff
@@ -38,8 +38,6 @@ export const DecodedData = ({ txData, toInfo }: Props): ReactElement | null => {
     )
   }
 
-  if (!txData) return null
```

Comment:
While writing tests I realised the only reason this condition is here is to satisfy Typescript so I refactored it. Now it will show that `txData` is defined further down.